pytests/stubs/pyclasses.pyi

```diff
@@ -18,6 +18,9 @@ class ClassWithDecorators:
 class ClassWithoutConstructor: ...
 
 class EmptyClass:
+    @classmethod
+    @property
+    def CLS_ATTRIBUTE(cls, /): ...
```

Comment:
I think `CLS_ATTRIBUTE: Final = 42` will work because `EmptyClass().CLS_ATTRIBUTE` works too.

We should maybe reuse the same code as for module constants.